local/bin/py/missing_metrics.py

```diff
@@ -0,0 +1,73 @@
+import datadog
+from datadog import initialize, api
+import time
+import os
+import glob
+import csv
+
+def get_csv_metrics(env):
+  csv_files = glob.glob(env + 'integration/*/*metadata.csv')
+
+  csv_metrics = {}
+  for csv_file in csv_files:
+    with open(csv_file, 'rb') as c_file:
+      csv_reader = csv.reader(c_file, delimiter=',')
+      for row in csv_reader:
+        if len(row) > 0:
+          if row[0] != 'metric_name':
+            csv_metrics[row[0]] = 1
+  return csv_metrics
+
+def get_dd_metrics(csv_metrics):
+  cloud = ('aws','azure', 'gcp') # tuple
+  ignore = ['isatap', '.p90', '.p95', '.p99', 'gcp.logging.user.', 'gcp.custom.']
+
+  # Datadog Demo account
+  options = {
+      'api_key': 'x', # dd-demo-api-key
+      'app_key': 'x' # dd-demo-app-key
+  }
+  initialize(**options)
+
+  # Taking the last:
+  days = 5
+  from_time = int(time.time()) - 60 * 60 * 24 * days
+
+  dd_metrics = api.Metric.list(from_time).get('metrics')
+  metrics_send = []
+  metrics_print = []
+
+  for metric in dd_metrics:
+    i = 0
+    if metric.startswith(cloud):
+      if metric not in csv_metrics:
+        for word in ignore:
+          if word in metric:
+            i = 1
+        if i == 0:
+          print(metric)
+          docs_tags = metric.split('.')
+          metrics_send.append({
+            'metric': 'docs.missing.metrics',
+            'points': 1,
+            'tags': ['docs_metric:' + metric, 'docs_cloud:' + docs_tags[0], 'docs_ns:' + docs_tags[1]]
+          })
+  print(len(metrics_send))
+  return metrics_send
+
+def post_dd_metrics(metrics):
+# Post to Corpsite Datadog account
+  options = {
+      'api_key': 'x', # dd-corp-api-key
+      'app_key': 'x' # dd-corp-app-key
+  }
+  datadog.initialize(**options)
+  print(datadog.api.Metric.send(metrics))
+
+if __name__ == '__main__':
```

Comment:
in here add something like this

```
from optparse import OptionParser
parser = OptionParser(usage="usage: %prog [options]")
parser.add_option("-k", "--apikey", help="api key", default=None)
parser.add_option("-p", "--appkey", help="app key", default=None)
options, args = parser.parse_args()
```

Then just pass the `options` variable to the 3 different functions below.

In those functions you should be able to reference the data by `options.appkey` or `options.apikey` etc.